### ISSUE TITLE
Remove aria-selected from accordion button

### ISSUE THREAD

**Issue by sarahpulis** [govau/design-system-components#595]
# Bug Report

- [x] I’ve read and understood the [Contributing guidelines](https://github.com/govau/design-system-components/blob/master/CONTRIBUTING.md) and have done my best effort to follow them.
- [x] I’ve read and agree to the [Code of Conduct](https://github.com/govau/design-system-components/blob/master/CODE_OF_CONDUCT.md).
- [x] I’ve searched for any related issues and avoided creating a duplicate issue.

## What happened

As per the [ARIA accordion design pattern](https://www.w3.org/TR/wai-aria-practices-1.1/#accordion), aria-selected is unnecessary on the accordion button. The aria-expanded attribute convey whether the accordion is expanded or collapsed.

**Comment by alex-page** [comment]
@sarahpulis This should be fixed on https://auds.service.gov.au/. We are publishing new documentation today for the design system site that should remove the incorrect tag.

I will close this issue when the documentation is updated. Thank you!

**Comment by alex-page** [comment]
This has now been resolved, please see the updated version and documentation:
https://designsystem.gov.au/components/accordion/
